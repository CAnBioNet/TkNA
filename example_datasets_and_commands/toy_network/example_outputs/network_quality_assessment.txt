PUC: 0.0%
Mean degree: 2.2857142857142856
Edges in network over the edges in a full graph: 0.38
Positive:negative edge ratio: 1.0

Expected positive:negative edge ratio in full graph: 0.75
Deviation from expected positive:negative edge ratio in full graph: 0.33

Number of positive nodes: 4
Number of negative nodes: 3
Number of total nodes: 7

Number of positive edges: 4
Number of negative edges: 4
Number of total edges: 8
