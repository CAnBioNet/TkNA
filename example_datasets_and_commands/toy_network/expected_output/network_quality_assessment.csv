Property,gene<==>pheno,pheno<==>pheno,gene<==>gene
PUC (%),0.0,0.0,0.0
Mean degree,1.0,2.0,2.0
Edges in network over edges in full graph (%),NA,NA,NA
Observed positive:negative edge ratio,NA,1.0,0.5
Expected positive:negative edge ratio,NA,NA,NA
Deviation from expected positive:negative edge ratio in full graph,NA,NA,NA
Positive nodes,2,2,2
Negative nodes,0,2,1
Total nodes,2,4,3
Positive edges,1,2,1
Negative edges,0,2,2
Total edges,1,4,3
